student.rb

```diff
@@ -3,8 +3,8 @@
 class Student < Person
   attr_reader :classroom
 
-  def initialize(classroom, age, name, parent_permission)
-    super(age, name, parent_permission)
+  def initialize(classroom:, age:, name:, parent_permission:)
```

Comment:
![er](https://github.com/mohamedgamaleldin999999/School-library/assets/89055502/db8b2129-aa34-4699-9114-35afca827d59)
- [ ] Your project throws up the below error when i try to add a new student and having inspected your code this highligted line is where you are having errors. the reason you are having this error is because  you passed in some parameters to your initialize constructor function like this `classroom:, age:, name:, parent_permission:`, doing this translates to `age: age, classroom: classrom, name: name, parent_permission: parent_permission` meaning you are setting a default value with the variable name use keyword arguments such as  `age, name, classroom, parent_permission` but there is no where in your code that age is defined as as such this is an error in your code. see the code snippet below for how it should be
- code snippet
```ruby
def initialize(classroom, age, name, parent_permission)
end
```
- [ ] Please i kindly advice you make this change in other places where you have defined the initialize constructor so that your code doesn't throw up any more errors and ultimately test that everything works before resubmitting :+1: :sparkles: